app/views/result.py

```
@@ -174,6 +174,10 @@ def result_view(year: int):
                 }
             )
 
+        # 各ラウンドの結果をrankで昇順ソート（1位から順に）
```

Ruff (0.14.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)